config/job.js

```diff
@@ -49,6 +52,8 @@ const SCHEMA_STEPS = Joi.object()
             }
         }
     });
+const SCHEMA_STEP = Joi.alternatives().try(SCHEMA_STEP_STRING, SCHEMA_STEP_OBJECT);
```

Comment:
So step can still be just a string like `npm publish` 
What will the validator output be for the `name`?